rasa/__main__.py

```diff
@@ -3,6 +3,7 @@
 import os
 import platform
 import sys
+import inspect
```

Comment:
this import is unused

Reply:
@Imod7 This is still here?

Reply:
Yes, I was waiting for feedback on the other 2 comments and then I would push. I will do a final check and push now.

Reply:
👍🏻 Could please also resolve the comments further up in the PR which you've worked in? Makes it easier to see which changes weren't addressed yet

Reply:
Yes! I will change the description of the PR and convert it from draft also.